python/pyspark/sql/tests/test_catalog.py

```diff
@@ -53,6 +53,14 @@ def test_database_exists(self):
             self.assertTrue(spark.catalog.databaseExists("spark_catalog.some_db"))
             self.assertFalse(spark.catalog.databaseExists("spark_catalog.some_db2"))
 
+    def test_get_database(self):
```

Comment:
No need to add a freshtable API test on Python side?

Reply:
done